samples/core/src/main/resources/config/application-mysql.properties

```diff
@@ -0,0 +1,9 @@
+spring.jpa.database=mysql
+spring.datasource.url=jdbc:mysql://localhost:3306/restaurant?user=root&password=mariaDB&connectTimeout=10000&socketTimeout=10000&autoReconnect=true
```

Comment:
equals to the other PR, these parameters should not be here. In my opinion we should keep them commented for reference/documentation purposes